src/js/adminSystem.js

```diff
@@ -126,9 +126,14 @@ function System(main) {
             for (var repo in that.systemRepos.native.repositories) {
                 if (!that.systemRepos.native.repositories.hasOwnProperty(repo)) continue;
                 var obj = that.systemRepos.native.repositories[repo];
-
+                let name = repo;
+                if (repo === 'default') {
+                    name = 'Stable (default)';
+                } else if (repo === 'latest') {
+                    name = 'Beta (latest)';
+                }
```

Comment:
I think this comparison should be extracted to a function, since you're using it three times.